### DISCUSSION TITLE
pangeo-builder fails to launch

### DISCUSSION THREAD

**Discussion by aparnaraveendran** [NCAR/cesm-lens-aws#69] [Q&A]
I am a research student, interested to use this data for research purposes.  I tried to re-create the notebooks in Pangeo-binder several times.  But it takes a long time to start the session and stops with an error.

The build log shows :
Launching attempt 1 failed, retrying...
Launching attempt 2 failed, retrying...
Launching attempt 3 failed, retrying...
Internal server error

What can be the issue?

**Comment by andersy005** [comment]
Thank you for the report, @AparnaRaveendran! I will look into this and will get back to you once I have a solution

**Reply by aparnaraveendran** [reply]
Thank you for the quick reply. I’ll wait to hear back from you.

**Reply by andersy005** [reply]
I've been trying to launch binder myself without success so far. I keep getting this:

<img width="1396" alt="Screen Shot 2021-01-21 at 8 43 30 PM" src="https://user-images.githubusercontent.com/13301940/105512478-62f0ed00-5c8e-11eb-9c82-20106b673378.png">

@scottyhq, do you happen to know what could be going on with https://aws-uswest2-binder.pangeo.io/? 

This is the binder link I am using: https://aws-uswest2-binder.pangeo.io/v2/gh/NCAR/cesm-lens-aws/binder-config?urlpath=git-pull?repo=https://github.com/NCAR/cesm-lens-aws%26amp%3Bbranch=master%26amp%3Burlpath=lab/tree/cesm-lens-aws/%3Fautodecode

**Reply by aparnaraveendran** [reply]
It would be very helpful if the binder link worked. I am really looking forward to work on this data.  
Thanks.

**Comment by scottyhq** [comment]
@andersy005 and @AparnaRaveendran ultimately the binder link in this repo and (.binder) config folder might need updating for compatibility with dask-gateway 0.9, but right now the Pangeo AWS binder is disabled (https://github.com/pangeo-data/pangeo-binder/issues/188). 

@AparnaRaveendran  in the meantime you can sign up for Hub access here for research use and run the notebook in this repository. https://pangeo.io/cloud.html#sign-up

**Reply by andersy005** [reply]
Thank you for the update, @scottyhq! I will be monitoring those discussions you linked to and will update the binder link and binder config once the binder is back online.

**Reply by aparnaraveendran** [reply]
@scottyhq That is very helpful. Thanks.
